test/rules/cascade_invocations.dart

```diff
@@ -40,9 +59,10 @@ class Foo {
 void noCascadeWithGetter() {
   final foo = new Foo();
   foo.baz(); // LINT
-  foo.bar; // LINT
-  foo.foo(); // LINT
+  foo.bar; // OK
+  foo.foo(); // OK
   foo.bar = 8; // LINT
```

Comment:
What's the rationale for producing a lint on a setter invocation when we don't producing a lint on a getter invocation? (And why do we not lint for a getter invocation?)